app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("app/cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            f" VALUES (?, ?)",
+            (first_name, last_name)
+        )
```

Comment:
Using f-strings to construct SQL queries can lead to SQL injection vulnerabilities. Although parameters are used here, ensure that all dynamic parts of the query are parameterized to avoid potential security issues.